docs/metrics.md

```diff
@@ -109,6 +114,7 @@ local schema = {
     {"object",                     "VARCHAR",   255,     nil,         "payload[object]"},
     {"method",                     "VARCHAR",   255,     nil,         "payload[method]"},
     {"domain",                     "VARCHAR",   255,     nil,         "payload[domain]"},
+    {"played_count",               "VARCHAR",   255,     nil,         "payload[played_count]"},
 
     {"video_x",                    "BOOLEAN",   nil,     nil,         "payload[video_x]"},
     {"video_y",                    "BOOLEAN",   nil,     nil,         "payload[video_y]"},
```

Comment:
Now that I look at this file, it seems like a mistake that these video_* keys are set to be a Boolean type. Would you mind updating these to be numbers?